doctrine/doctrine-bundle/2.3/config/packages/doctrine.yaml

```diff
@@ -0,0 +1,19 @@
+doctrine:
+    dbal:
+        override_url: true
```

Comment:
I don't think we need this here... just because the user won't be overriding anything.

But what we *should* have, I think, is a `config/packages/test/doctrine.yaml` that has this setting. AND... I'm tempted to actually set the `dbname: ` there it something... maybe even something that has `TEST_TOKEN` in it for Paratest.

OR, we could actually do nothing (not even make any changes to the recipe) and instead we rely on documentation to guide the user to create the `config/packages/test/doctrine.yaml` file.

Reply:
not setting `override_url: true` while using a `url` triggers the new deprecation in the bundle.

`Not setting doctrine.dbal.override_url to true is deprecated. True is the only value that will be supported in doctrine-bundle 3.0.`

Reply:
duh. I totally forgot :p.

But should we also have a `config/packages/test/doctrine.yaml` file? I kind think we should...

Reply:
done.


>maybe even something that has TEST_TOKEN in it for Paratest.


I'm not familiar with Paratest, but I gather the `TEST_TOKEN` has to be unique in every test? I don't see how we could achieve that in the config - but I am probably missing something obvious.

Reply:
`TEST_TOKEN` is *provided* by ParaTest - they set it automatically. All we need to do is *read* that environment variable with a default value. If it's not there, then it would be empty and... who cares :). The idea would be that we have something like `main_test%env('TEST_TOKEN')%`